manifests/base/base-rhel9-imagestream.yaml

```diff
@@ -0,0 +1,16 @@
+---
```

Comment:
we dont require manifest files for this ,
please revert the imagestream change for base-rhel9

remove the changes under  manifests/base

Reply:
removed